.github/workflows/deploy-latest.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy latest docker.
+
+on:
+  push:
+    branches:
+      - "master"
+      - "staging"
```

Comment:
If I understand correctly this worfklow, this builds then pushes the `arnavon:latest` docker image. We don't want to do that on both "master" and "staging" branches. Only on "master".

Reply:
Ok.